concent_api/core/tests/test_unit_validation.py

```diff
@@ -62,3 +64,27 @@ def test_that_function_raise_http400_when_price_is_negative(self):
 
     def test_that_function_will_not_return_anything_when_price_is_zero(self):  # pylint: disable=no-self-use
         validate_subtask_price_task_to_compute(mocked_message_with_price(0))
+
+
+class TestValidatorReportComputedTaskList(ConcentIntegrationTestCase):
+
```

Comment:
I like how neat and precise the tests are :+1: 
One problem though is naming of the tested function (please see: my comment below)